### DISCUSSION TITLE
webui for bash (webui.sh)

### DISCUSSION THREAD

**Discussion by Anyeos** [hlky/stable-diffusion-webui#710] [Show and tell]
Hi, I have developed a basic webui.sh (bash based) to run like the cmd version. 
Take in account that it is basic (but is working). I don't know if I will have the time to improve it, but it really does not require too much time to improve so maybe in a time I will do.

For now, you must specify the custom_conda_path path manually editing the file. And don't forget to review "conda_env_name" too.
[webui.zip](https://github.com/sd-webui/stable-diffusion-webui/files/9500827/webui.zip)

**Comment by grmrgecko** [comment]
Consider submitting a pull request, may help others.

**Reply by grmrgecko** [reply]
Actually, I'll do it after porting more of the functionality from the windows script.

**Reply by grmrgecko** [reply]
Pull request is now at https://github.com/sd-webui/stable-diffusion-webui/pull/723
